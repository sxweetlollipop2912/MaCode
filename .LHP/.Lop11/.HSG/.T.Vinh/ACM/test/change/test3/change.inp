50
klhliiglgikijgjkjggl
gggggggggggggggggggg
lllllllklglkkklkkggg
mgkllnlmmngjlngmnlng
dfhhdgihdffhfdifihdi
daaabdabdbadcdadddba
aaaaacaaaaaaaaaaaaaa
hjihiimhhnmjjmmkmkim
trsotsroutrptrsssttq
cecceccgccccedeeecdc
ggacadagggagadgdfgcg
mnmnmmmmlljmjjqqnnln
idaiigcddcgaaaifaiih
cbacbabbbaacccbcaacb
cddgbadbdcbbbbcdadcb
hhdghgcagdabaebaghab
fcdcdfcbdfdffddedfed
cdcegggcdgaggdeceadc
cccccccccccccccccccc
bbbabababbacccbbaabb
ppnpmnmqmonmqmpnrmpm
kkkkkkkkkkkkkkkkkkkk
hlgmjmmmgggggklglglm
gkgggiggkggilgigkggg
dadaadadaaaaabaadadd
mjjklggkjogokjkgkjoo
aaacaaaabaaaacaaaaaa
ogoihggkgggeeceioggl
kkkkkkjkkkkkkkkkkjjj
jjgjjjjgjgggjjhgjjgg
ghkggggghggjkgkkggij
kokkkooopnkookooopol
acbccacbbcbaaacacbcb
lkhilklklhklkkhkkkhl
cdcffeebfcecfefceeef
gkgkkkkkgggggkkhkikg
oooooolonloojoooojjn
gkgkggklggkggkgklgkg
sosnvqonosurorusosrq
npkkpppppnkkpkpknnpl
llllllllllllllllllll
orpqnnrkknjpjjpjpprn
ijhijiiijhijjijhjkij
mlomqmmjqmpooljmlqmj
mpkiomjppppmpoppommp
dedefdfefefffbeedffd
cdhfdccgccccccgccccg
caacdabbcbadbacbcbab
kmrnmkrkornqknpppprk
aaaaaaaaaaaaaaaaaaaa
