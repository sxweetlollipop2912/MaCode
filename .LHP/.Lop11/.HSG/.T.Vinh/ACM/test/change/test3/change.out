19 46
18 23
19 46
19 48
11 36
2 21
2 17
21 48
41 50
8 22
2 22
24 50
2 36
2 17
2 22
2 33
2 22
2 22
7 18
2 17
36 50
28 43
19 47
19 46
2 21
19 49
2 17
8 49
24 42
19 37
19 42
29 50
2 17
21 46
2 22
19 42
24 49
19 46
38 50
29 50
34 47
24 50
21 42
24 50
23 50
2 22
8 33
2 21
29 50
1 1
